products/chef-infra-client.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Chef Infra Client
+category: server-app
```

Comment:
Is this the appropriate category for the client?

Reply:
moved to "app"